Lambdas/GHOAuth/OAuthLambda.swift

```diff
@@ -38,10 +38,10 @@ struct GHOAuthHandler: LambdaHandler {
     }
 
     init(context: LambdaInitializationContext) async throws {
-        self.client = HTTPClient(eventLoopGroupProvider: .shared(context.eventLoop))
+        self.client = HTTPClient.shared
```

Comment:
Why do you even need a `client` property at all at this point?

Reply:
I did think about it ... was kind of scared to remove all of them but I still can't see why Penny would need them ...